index.js

```diff
@@ -11,7 +11,10 @@ module.exports = function (options) {
     var cmdArgs = ["restore"];
 
     if (options) {
-        nugetPath = options.nugetPath || nugetPath;
+        if (options.nugetPath && options.nugetPath.length > 0) {
```

Comment:
I don't believe you need to check for the length being greater than zero. Empty strings count as being false